src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+@import "./utils/variables";
+@import "./utils/reset";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap;
+  padding: 20px $padding;
+  font-size: 30px;
+
+
+  &__day {
+    display: flex;
+    height: $day-size;
+    width: $day-size;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    border: $day-border solid $c-border;
+    background-color: $c-day;
+    width: $day-size;
+    height: $day-size;
+    gap: $gap;
+    transition: color 0.5s, transform 0.5s;
+  }
+    & :hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+
+  @for $day from 1 through 31 {
+      &__day:nth-child(#{$day})::before {
+        content: "#{$day}";
+      }
+  }
+
+$week-value:
```

Comment:
as for me, would be better to move it to variables.scss. In the future, it will be easier to maintain this code.